proposals/revised-governance.md

```diff
@@ -0,0 +1,332 @@
+
+The CTC evolves into the Core Working Group. It would still define it’s own
+processes and its own meeting schedule.
+
+### Why are working groups required to have someone on the Admin team?
+
+This is to ensure proper representation among the working groups when key
+decisions are made with regards to budget allocation and administrative actions
+such as moderation enforcement. Each working group is free to determine how it
+chooses the individual to sit on the Admin team, and may choose to opt out of the
+responsibility when the working group is chartered. A working group that has
+opted out may ask to join the Admin team at any time, and it would be up to the
+existing members of that WG to decide the membership policy.
+
+### Why is the TSC Director the chair of the Admin team?
+
+This is primarily for logistical reasons. The Admin team will be responsible for
+dealing with budget allocations and administrative tasks that involve
+interfacing with the Node.js Foundation. The TSC Director will be the primary
+touch point for such activities. This also echoes the situation in the
+Community Committee where the individual member representatives on the board
+share the Community Committee chair responsibilities.
+
+### This takes some power away from the CTC!
+
+Yes and no. The Core Working Group (or what we call the CTC today) would still
+have overall responsibility for nodejs/node for any aspect that is not
+specifically delegated to another working group (like Streams, for instance).
+
+For items that are under Core's responsibility, the Working Group would still
+retain the ability to make decisions following whatever process it chooses
+(likely no different than the process we have today in the CTC). However, for
+areas that have been delegated to other working groups, those working groups
+make the decisions, with any working group member having the ability to
+escalate any discussion/decision to `tsc-review`.
+
+### It will take longer to get things done!
+
+I disagree. I shouldn't take any longer than it does today.
+
+The goal is for working groups to be more empowered to make decisions on their
+own as much as possible. There will be less need to push decisions to the CTC
+and TSC and therefore less process around making decisions.
+
+### This just means we’ll have more Working Groups!
+
+I disagree. Working groups will carry with them a certain amount of required
+process. Chartering a new working group would also require consensus of all the
+other working groups, which means that there would be a fairly high bar to
+adding a new working group. This would mean that the growth of the number of
+voting seats would be much slower and manageable than the current potentially
+unbounded growth.
+
+### One monthly TSC meeting in which anyone can participate sounds like chaos!
+
+It will require better logistical planning, yes, but other groups (such as
+TC-39 for instance) pull off such meetings with very little difficulty. It
```

Comment:
It is not guaranteed to however. An increased amount of politics from WGs _could_ end up in increased off-github backchanneling. We just need to be aware of that this could happen.